typesafe_conductr_cli/conduct_load.py

```diff
@@ -8,12 +9,16 @@
 def load(args):
     """`conduct load` command"""
 
+    if args.bundle_name is None:
+        args.bundle_name = "-".join(os.path.basename(args.bundle).split("-")[:-1])
```

Comment:
Okay. Will add a check that looks for the hex string at the back and strips that.